### PR TITLE
docs: add guide on integrating CMake

### PR DESCRIPTION
**Pull request by Panquesito7**
#### Description of Change

- Add a guide on integrating CMake in new folders.

#### Checklist

- [x] Added description of change
- [ ] ~Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#File-Name-guidelines)~
- [ ] ~Added tests and example, test must pass~
- [ ] ~Relevant documentation/comments is changed or added~
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -150,9 +150,46 @@ some_new_fancy_category         is correct
...
+```cmake
+...
+add_subdirectory(divide_and_conquer)
+add_subdirectory(<foldername>)
+```
+
 #### Commit Guidelines
 
-- It is recommended to keep your changes grouped logically within individual commits. Maintainers find it easier to understand changes that are logically spilt across multiple commits. Try to modify just one or two files in the same directory.  Pull requests that span multiple directories are often rejected.
+- It is recommended to keep your changes grouped logically within individual commits. Maintainers find it easier to understand changes that are logically spilled across multiple commits. Try to modify just one or two files in the same directory. Pull requests that span multiple directories are often rejected.
```
Comment (tjgurwara99) [inline CONTRIBUTING.md:192]:
```suggestion
- It is recommended to keep your changes grouped logically within individual commits. Maintainers find it easier to understand changes that are logically split across multiple commits. Try to modify just one or two files in the same directory. Pull requests that span multiple directories are often rejected.
```

I think it was correct...

**Review by tjgurwara99 — CHANGES_REQUESTED** [review]
🚀 Consider the some of the following minor comments 😄

Also, if possible could you split the lines and make them shorter? It helps keep the diffs smaller, easier to review and more concise 😄

File: CONTRIBUTING.md (hunk 1 above)
Comment (Panquesito7) [reply CONTRIBUTING.md:192]: Oh, I think `split` is American English, and `spilled` is British English. Which one should we use?

Comment (tjgurwara99) [reply CONTRIBUTING.md:192]: Ah I think that's not quite right 😅 `split` means "breaking things apart" but `spilled` means (when talking about liquid) causing something to go over flow of container (something along those lines) 😄

Comment (Panquesito7) [reply CONTRIBUTING.md:192]: Hmm. Should I keep it as-is or change it back to `spilled`?

Comment (tjgurwara99) [reply CONTRIBUTING.md:192]: Keep it as is 😄

**Review by tjgurwara99 — APPROVED** [review]

**Comment by tjgurwara99** [comment]
I have also opened a [PR to fix the workflows](https://github.com/TheAlgorithms/C/pull/1176) failing - so you can ignore those current failures rn.

**Pull request merged**